Server/Common/Common.Domain/ErrorMessages.cs

```diff
@@ -32,4 +34,13 @@ public static class ErrorMessages
     }
 
     public const string InternalServer = "Internal server error.";
+
+    public static string NotSuccessResponse(HttpMethod method, string requestUri) =>
+        $"HTTP {method.Method} - {requestUri} responded with not success StatusCode.";
```

Comment:
in this method the message begins with HTTP, but method name doesn't contains any information about HTTP.